--- origsrc/kmix-14.12.3/CMakeLists.txt	2014-10-21 18:27:24.000000000 -0500
+++ src/kmix-14.12.3/CMakeLists.txt	2015-05-11 19:02:06.173134100 -0500
@@ -242,7 +242,7 @@ if (CANBERRA_FOUND)
   target_link_libraries(kdeinit_kmix ${CANBERRA_LIBRARIES})
 endif (CANBERRA_FOUND)
 
-install(TARGETS kdeinit_kmix  DESTINATION ${LIB_INSTALL_DIR} )
+install(TARGETS kdeinit_kmix ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 target_link_libraries( kmix kdeinit_kmix )
 install(TARGETS kmix  ${INSTALL_TARGETS_DEFAULT_ARGS} )
@@ -375,7 +375,7 @@ endif()
 ####################################################################################################
 ########### INSTALL RULES ##########################################################################
 ####################################################################################################
-install( TARGETS kdeinit_kmixctrl  DESTINATION ${LIB_INSTALL_DIR} )
+install( TARGETS kdeinit_kmixctrl ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 target_link_libraries( kmixctrl kdeinit_kmixctrl )
 install( TARGETS kmixctrl ${INSTALL_TARGETS_DEFAULT_ARGS} )
