--- origsrc/kdemultimedia-4.7.0/kmix/CMakeLists.txt	2011-06-22 15:40:35.000000000 -0500
+++ src/kdemultimedia-4.7.0/kmix/CMakeLists.txt	2011-08-08 02:49:32.630640700 -0500
@@ -82,7 +82,7 @@ if (HAVE_PULSE)
 endif (HAVE_PULSE)
 
 
-install(TARGETS kdeinit_kmix  DESTINATION ${LIB_INSTALL_DIR} )
+install(TARGETS kdeinit_kmix ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 target_link_libraries( kmix kdeinit_kmix )
 install(TARGETS kmix  ${INSTALL_TARGETS_DEFAULT_ARGS} )
@@ -155,7 +155,7 @@ endif (HAVE_PULSE)
 ########### next target ###############
 add_subdirectory( plasma )
 
-install( TARGETS kdeinit_kmixctrl  DESTINATION ${LIB_INSTALL_DIR} )
+install( TARGETS kdeinit_kmixctrl ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 target_link_libraries( kmixctrl kdeinit_kmixctrl )
 install( TARGETS kmixctrl ${INSTALL_TARGETS_DEFAULT_ARGS} )
